PSOperations/RetryOperation.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RetryOperation.swift
+//  PSOperations
+//
+//  Created by Bart Whiteley on 9/22/15.
+//
+
+
+public class RetryOperation: GroupOperation {
+    private let operationProducer: () -> Operation
+    private let shouldRetry: (errors:[NSError], retryCount:Int) -> Bool
+    private var numberOfRetries: Int = 0
+    
+    public convenience init(@autoclosure(escaping) operation: () -> Operation, maximumNumberOfRetries: Int = 5) {
+        self.init(operationProducer: operation, shouldRetry: { (_, retries) in return retries < maximumNumberOfRetries })
```

Comment:
I agree that's how it should work. But looking at the code here, the `shouldRetry` doesn't use the errors. It only tests `retries < maximumNumberOfRetries`